README.md

```diff
@@ -63,6 +75,7 @@ gem install bundler
 ```
 bin/setup
 ```
+* Rename `env-example` file to `.env`
```

Comment:
doesn't bin/setup do this already here https://github.com/ServiceInnovationLab/RapuTure/blob/98df8b6371111cd7d1b939bab70cb863274c741c/bin/setup#L25